docs/markdown/snippets/response-files.md

```diff
@@ -0,0 +1,4 @@
+## Response files when needed
+
+Meson now uses response files on more platforms to avoid commandline
```

Comment:
This makes it sound like response files are used unconditionally.

Reply:
Fixed.